web_src/js/components/DiffFileTreeItem.vue

```diff
@@ -10,7 +10,7 @@
       />
       <a
         v-if="item.isFile"
-        class="file gt-ellipsis"
+        :class="['file', 'gt-ellipsis', item.file.IsViewed ? 'viewed' : 'not-viewed']"
```

Comment:
```suggestion
        :class="['file gt-ellipsis', {'viewed': item.file.IsViewed}]"
```

`not-viewed` is not used